libs/@guardian/source/src/react-components/__generated__/icons/SvgAlarmClockFilled.tsx

```diff
@@ -1,9 +1,8 @@
 // DO NOT EDIT
 // this file is auto-generated by libs/@guardian/source/scripts/create-icons/index.ts
 import { css } from '@emotion/react';
-import type { IconProps } from '../..';
 import { iconSize, visuallyHidden } from '../../../foundations';
-
+import type { IconProps } from '../..';
```

Comment:
Do you know why this ordering has changed across so many files? It seems odd, unless a lint rule has changed?